Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragment.kt

```diff
@@ -46,10 +32,12 @@ class OnboardingFragment : Fragment() {
     private fun setLinks() {
         binding.onboardingInclude.onboardingEasyLanguage
             .setOnClickListener {
-            val browserIntent = Intent(Intent.ACTION_VIEW,
-                Uri.parse(getString(R.string.onboarding_tracing_easy_language_explanation_url)))
-            startActivity(browserIntent)
-        }
+                val browserIntent = Intent(
```

Comment:
`val` not needed, Intent could be created within startActivity

Reply:
Okay in this case, otherwise we have double (()) nesting again, could go for kotlin `.also`  but none of that is really an improvement.